build.properties

```diff
@@ -2,10 +2,8 @@
 # Don't directly modify this file. Instead, copy it to local.build.properties and
 # edit that.
 #
-# $Id$
 project.name = eXist-db
-project.version = 3.0
```

Comment:
@adamretter we agree to have single version, but we did not have any agreement that it going to be `project.version`.

You are right that extensions should know what development stage, but human that use eXist must know that. And that's why distribution must clear declare development stage information.